src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -771,6 +778,14 @@ public void close()
 
         if (deviceSession != null)
             deviceSession.close();
+
+        RTCPTerminationStrategy strategy
+            = rtcpTransformEngineWrapper.getWrapped();
+        if (strategy instanceof RecurringRunnable)
+        {
+            recurringRunnablesExecutor
+                .deRegisterRecurringRunnable((RecurringRunnable) strategy);
+        }
```

Comment:
I agree this is disturbing. To quote @bgrozev from the commit message:

>> Closes #cachingTransformer on MediaStreamImpl#close(), fixing a leak when jitsi-videobridge performs health checks. We cannot rely on it being closed with #transformEngineChain, because the chain isn't always initialized (when helath-checks are run on jitsi-videobridge, it creates MediaStreams but never sets their RTPConnector, so the chain is never created.

Can we leave it as is for now, or you think this is completely unacceptable?